### PR TITLE
Internal: Fix CI deprecation warnings [ED-14058]

### PR DESCRIPTION
**Pull request by dmaziar**
## PR Checklist

- [ ] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

### CODE REVIEW

**Review by yotamselementor — CHANGES_REQUESTED** [review]

File: .github/workflows/playwright.yml [hunk 1]
```diff
@@ -116,30 +106,25 @@ jobs:
...
+      - name: Install dependencies
+        run: npm ci
       - name: Restore build from cache
-        uses: actions/cache/restore@v3
+        uses: actions/cache/restore@v4
         id: restore-build
         with:
           path: ./build/*
-          key: playwright-build-${{ github.sha }}
+          key: playwright-build123-${{ github.sha }}
```
[the remark this answers is not in the document]
Comment (dmaziar) [reply .github/workflows/playwright.yml:122]: yes, some other job stores the same build, so cache save is failing

**Review by yotamselementor — COMMENTED** [review]
Notice that you did not upgrade all scripts to use. Node 20.X

**Comment by dmaziar** [comment]
> Notice that you did not upgrade all scripts to use. Node 20.X

fixed

File: .github/workflows/playwright.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (yotamselementor) [reply .github/workflows/playwright.yml:122]: then maybe 
```suggestion
          key: playwright-build-v4-${{ github.sha }}
```

[the remark this answers is not in the document]
Comment (dmaziar) [reply .github/workflows/playwright.yml:122]: fixed

**Review by yotamselementor — DISMISSED** [review]

**Review by yotamselementor — APPROVED** [review]

**Pull request merged**